dipy/reconst/bingham.py

```diff
@@ -0,0 +1,256 @@
+"""Tools for fitting Bingham distributions to orientation distributions,
+as described in Riffert et al [1]_. The resulting distributions can further
+be used to compute ODF-lobe-specific measures such as the fiber density
+and fiber spread [1]_.
+
+References
+----------
+.. [1] Riffert TW, Schreiber J, Anwander A, Knösche TR. Beyond fractional
+           anisotropy: Extraction of bundle-specific structural metrics from
+           crossing fiber models. NeuroImage. 2014 Oct 15;100:176-91.
+"""
+
+from math import cos, radians
+from warnings import warn
+import numpy as np
+
+from dipy.direction import peak_directions
+
+
+def bingham_to_sf(f0, k1, k2 , major_axis, minor_axis, vertices):
```

Comment:
Can you change this function's name.? It's a bit confusing because SF is too close to FS which is the name of one of the output quantities. We suggest changing it to `bingham_odf`, because this function computes an ODF based on a Bingham function.